### ISSUE TITLE
Make it able to load ”relaxed” YAML files

### ISSUE THREAD

**Issue by ralienpp** [mk-fg/pretty-yaml#9]
I wrote a simple wrapper around the yaml library that gives me some additional features:
- comments
- inline comments
- warnings about mixed spaces and tabs
- warnings about lines that contain ambiguous `#` characters (could be comment or a URL anchor)

I use it for loading configuration files for my systems. Comments are really helpful for admins who run these systems, as the config file also acts as a source of documentation.

Basically, it is the same philosophy, but aimed at the opposite end - reading YAML files written in a humane fashion.

Would you agree with adding this relaxed reading feature to this module? Here's a draft: https://github.com/ralienpp/pretty-yaml/commit/cd7de496539204247b6775e295caa68f685e9bd4
